lib/msal-common/src/authority/OpenIdConfigResponse.ts

```diff
@@ -3,9 +3,12 @@
  * Licensed under the MIT License.
  */
 
+import { ICloudDiscoveryMetadata } from "./ICloudDiscoveryMetadata";
+
 /**
  * The OpenID Configuration Endpoint Response interface. Used by the authority class to get relevant OAuth endpoints.
  */
 export interface OpenIdConfigResponse {
```

Comment:
name is throwing me. This is not really `OpenIdConfigResponse`, can we please change it?

Reply:
Done